examples/align-advanced/CMakeLists.txt

```diff
@@ -2,18 +2,13 @@
 # Copyright(c) 2019-2024 Intel Corporation. All Rights Reserved.
 cmake_minimum_required(VERSION 3.8)
 
-project( rs-align-advanced )
-
-if( BUILD_GRAPHICAL_EXAMPLES )
-
-add_executable( ${PROJECT_NAME} rs-align-advanced.cpp
-    ../../third-party/imgui/imgui.cpp ../../third-party/imgui/imgui_draw.cpp ../../third-party/imgui/imgui_impl_glfw.cpp
-    )
-set_property( TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11 )
-target_link_libraries( ${PROJECT_NAME} ${DEPENDENCIES} )
-include_directories( ../../third-party/imgui ../../examples )
-set_target_properties( ${PROJECT_NAME} PROPERTIES FOLDER Examples )
-using_easyloggingpp( ${PROJECT_NAME} SHARED )
-install( TARGETS ${PROJECT_NAME} RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} )
-
+project(RealsenseExamplesAlignAdvanced)
+
+if(BUILD_GRAPHICAL_EXAMPLES)
+    add_executable(rs-align-advanced rs-align-advanced.cpp ../../third-party/imgui/imgui.cpp ../../third-party/imgui/imgui_draw.cpp ../../third-party/imgui/imgui_impl_glfw.cpp  ../../third-party/imgui/realsense_imgui.cpp ../../third-party/imgui/imgui_impl_opengl3.cpp ../../third-party/imgui/imgui_widgets.cpp ../../third-party/imgui/imgui_tables.cpp)
+    set_property(TARGET rs-align-advanced PROPERTY CXX_STANDARD 11)
+    target_link_libraries(rs-align-advanced ${DEPENDENCIES})
+    include_directories(../../common ../../third-party/imgui ../../examples)
+    set_target_properties (rs-align-advanced PROPERTIES FOLDER Examples)
+    install(TARGETS rs-align-advanced RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
Where does 
'using_easyloggingpp' disappeared? is it needed?
Maybe rebase will fix it

Reply:
rebase fixed it